swagger.yml

```diff
@@ -870,7 +870,7 @@ paths:
   /cars/search-airport:
     get:
       operationId: Car Rental Airport Search
-      summary: The Car Rental API uses Amadeus' classic car service to locate the best available car. Define a circular area with one coordinate and radius, or provide an airport code, as well as the pick-up and drop-off dates, and get a list of car rental providers with their locations and rates. Optional parameters such as currency and rental provider codes are also available and can be used to narrow down the results. This API is an ideal pairing with the flight and hotel search to provide ground transportation options at the destination.
+      summary: Car Rental Airport Search - Provide an airport code, as well as the pick-up and drop-off dates, and get a list of car rental providers with their locations and rates. Optional parameters such as currency and rental provider codes are also available and can be used to narrow down the results. This API is an ideal pairing with the flight and hotel search to provide ground transportation options at the destination.
       description: |
```

Comment:
Be consistent, either describe what the API does or tell the user how to use it.